### ISSUE TITLE
Fortran syntax file broken

### ISSUE THREAD

**Issue by sambo57u** [vim/vim#5350]
I am using vim under Fedora 31, current version 8.1.2424.

I am not sure exactly when but not so long ago everytime I edit a fortran file, I get the error messages
on the terminal from the fotrran.vim syntax file:

```
=======================================
'Error detected while processing /usr/share/vim/vim81/syntax/fortran.vim:
line  188:
E869: (NFA) Unknown operator '\@1'
E475: Invalid argument: fortranTarget^Idisplay^I"\(\<do\s\+\)\@<11=\d\+\>"
line  277:
E869: (NFA) Unknown operator '\@1'
E475: Invalid argument: fortranConstructName "\(\<end\s*forall\s\+\)\@<15=\a\w*\>"
==========================================
```

The issue seems to be the < after \@, e..g., \@<15=... If I remove these < signs then 
error goes away but I am not sure whether that is the right thing to do.

**Comment by chrisbra** [comment]
Yeah, this looks wrong:

https://github.com/vim/vim/blob/98056533b96b6b5d8849641de93185dd7bcadc44/runtime/syntax/fortran.vim#L188-L189

https://github.com/vim/vim/blob/98056533b96b6b5d8849641de93185dd7bcadc44/runtime/syntax/fortran.vim#L277

As a workaround, try not to define the variable `fortran_more_precise`

**Comment by chrisbra** [comment]
you could move the number before the `<`

**Comment by chrisbra** [comment]
Please contact the maintainer of the file mentioned at the top.

**Comment by sambo57u** [comment]
Thanks, will do!

**Comment by bytbox** [comment]
Checking [vim-dev], I see that the maintainer has sent a fix for the issue (off-list) to Bram. The fix doesn't appear to be in this repo yet. Is the fix available publicly, anywhere?

(@chrisbra's suggestion of just replacing `<15=` with `15<=` and `<11=` with `11<=` seems to work, but I really don't know. At least it gets rid of the error messages.)

**Comment by vim-ml** [comment]
I just resent the updated file to Bram.

Cheers
Ajit

**Comment by chrisbra** [comment]
closing then.

**Comment by bytbox** [comment]
I hate to be too much of a pest, but as this issue is not fixed in the repo, and the patch is not publicly available, I would suggest re-opening the issue and leaving it so for the time being.

**Comment by k-takata** [comment]
I believe the change is already included in Bram's local repository and will be published in next runtime files update. Let's wait for it.

**Comment by brammool** [comment]
> I believe the change is already included in Bram's local repository and
> will be published in next runtime files update. Let's wait for it.

The version I have is from 2019 Dec 14.  Not aware of a newer one.

-- 
This planet has -- or rather had -- a problem, which was this: most
of the people living on it were unhappy for pretty much of the time.
Many solutions were suggested for this problem, but most of these
were largely concerned with the movements of small green pieces of
paper, which is odd because on the whole it wasn't the small green
pieces of paper that were unhappy.
		-- Douglas Adams, "The Hitchhiker's Guide to the Galaxy"

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by bytbox** [comment]
For clarity for anyone in the future: this issue is fixed by commit `560979ed...`.

Thanks all!
